site/docs/cquery.html

```diff
@@ -600,6 +600,22 @@ <h4>Examples</h4>
   $ bazel cquery //baz --output=starlark --starlark:file=example.cquery
 </pre>
 
+<p>
+  Filter out <a href="platforms.html#skipping-incompatible-targets">incompatible targets</a> and
+  only print compatible targets.
+</p>
+<pre>
+  $ cat example.cquery
+
+  def format(target):
+    if "IncompatiblePlatformProvider" not in providers(target):
+      return target.label
+    return ""
+
+
+  $ bazel cquery //... --output=starlark --starlark:file=example.cquery
+</pre>
+
```

Comment:
I could also move this into the Platforms page instead of here. I couldn't convince myself one way over the other.

Reply:
I'd move this to the platforms page, since `cquery` is just the tool that happens to serve the need.

Reply:
Sounds good to me. Done.